src/main/webapp/app/shared/components/documentation-button/documentation-button.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, Input } from '@angular/core';
+import { faCircleInfo } from '@fortawesome/free-solid-svg-icons';
+
+// The routes here are used to build the link to the documentation.
+// Therefore, it's important that they exactly match the url to the subpage of the documentation.
+// Additionally, the case names must match the keys in documentationLinks.json for the tooltip.
+export enum DocumentationType {
+    Course = <any>'courses/customizable/',
+    Lecture = <any>'lectures/',
+    Exercise = <any>'exercises/',
+    Quiz = <any>'exercises/quiz/',
+    Model = <any>'exercises/modeling/',
+    Programming = <any>'exercises/programming/',
+    Text = <any>'exercises/textual/',
+    FileUpload = <any>'exercises/file-upload/',
+    Notifications = <any>'notifications/',
+    LearningGoals = <any>'lectures/#learning-goals',
```

Comment:
As far as I'm aware, there is no #learning-goals section on the [lectures page](https://docs.artemis.cit.tum.de/user/lectures/#).